content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -103,97 +96,53 @@ dotnet run
 
 ## Instrumentation
 
-Next, you'll use a [OpenTelemetry .NET Automatic Instrumentation](../automatic)
-to instrument the application at launch time. While you can [configure .NET
-Automatic Instrumentation][] in a number of ways, the steps below use Unix-shell
-or PowerShell scripts.
-
-> **Note**: PowerShell commands require elevated (administrator) privileges.
-
-1. Download installation scripts from [Releases][] of the
-   `opentelemetry-dotnet-instrumentation` repository:
-
-   {{< tabpane text=true >}} {{% tab Unix-shell %}}
-
-   ```sh
-   curl -L -O https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/latest/download/otel-dotnet-auto-install.sh
-   ```
-
-   {{% /tab %}} {{% tab PowerShell - Windows %}}
-
-   ```powershell
-   $module_url = "https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/latest/download/OpenTelemetry.DotNet.Auto.psm1"
-   $download_path = Join-Path $env:temp "OpenTelemetry.DotNet.Auto.psm1"
-   Invoke-WebRequest -Uri $module_url -OutFile $download_path -UseBasicParsing
-   ```
-
-   {{% /tab %}} {{< /tabpane >}}
-
-2. Execute following script to download automatic instrumentation for your
-   development environment:
+Next we'll install the instrumentation nuget packages from OpenTelemetry that will generate the telemetry, and set them up.
 
-   {{< tabpane text=true >}} {{% tab Unix-shell %}}
+1. Add the packages
 
-   ```sh
-   ./otel-dotnet-auto-install.sh
-   ```
-
-   {{% /tab %}} {{% tab PowerShell - Windows %}}
-
-   ```powershell
-   Import-Module $download_path
-   Install-OpenTelemetryCore
-   ```
-
-   {{% /tab %}} {{< /tabpane >}}
-
-3. Set and export variables that specify a [console exporter][], then execute
-   script configuring other necessary environment variables using a notation
-   suitable for your shell/terminal environment &mdash; we illustrate a notation
-   for bash-like shells and PowerShell:
+```shell
+dotnet add package OpenTelemetry.Extensions.Hosting
+dotnet add package OpenTelemetry.Instrumentation.AspNetCore --prerelease
+dotnet add package OpenTelemetry.Exporter.Console
+```
 
-   {{< tabpane text=true >}} {{% tab Unix-shell %}}
+2. Setup the packages
 
-   ```sh
-   export OTEL_TRACES_EXPORTER=none \
-     OTEL_METRICS_EXPORTER=none \
-     OTEL_LOGS_EXPORTER=none \
-     OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED=true \
-     OTEL_DOTNET_AUTO_METRICS_CONSOLE_EXPORTER_ENABLED=true \
-     OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED=true
-     OTEL_SERVICE_NAME=RollDiceService
-   . $HOME/.otel-dotnet-auto/instrument.sh
-   ```
+```csharp
+var builder = WebApplication.CreateBuilder(args);
 
-   {{% /tab %}} {{% tab PowerShell - Windows %}}
+builder.Logging.AddOpenTelemetry(options =>
```

Comment:
attach resource to logging as well.

Reply:
`ConfigureResource` called on `builder.Services.AddOpenTelemetry()` should add the configured resource to the logger provider as well.

@CodeBlanch Could you please confirm?